app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -4,39 +4,293 @@
+                android:layout_alignStart="@+id/tvCast">
+
+                <ImageView
+                    android:id="@+id/cast1Img"
+                    android:layout_width="120dp"
+                    android:layout_height="wrap_content"
+                    android:layout_alignParentBottom="true"
+                    android:layout_marginTop="4dp"
+                    android:layout_marginBottom="0dp"
+                    tools:srcCompat="@tools:sample/avatars" />
+
+                <TextView
+                    android:id="@+id/cast1Name"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_alignTop="@+id/cast1Img"
+                    android:layout_marginStart="130dp"
+                    android:layout_marginTop="15dp"
+                    tools:text="Cast Member Name" />
+
+                <TextView
+                    android:id="@+id/cast1Role"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_alignTop="@+id/cast1Name"
+                    android:layout_marginStart="130dp"
+                    android:layout_marginTop="20dp"
+                    tools:text="Character Name" />
+
+            </RelativeLayout>
+
+            <RelativeLayout
+                android:id="@+id/cast2"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/cast1"
+                android:layout_alignStart="@+id/tvCast">
+
+                <ImageView
+                    android:id="@+id/cast2Img"
+                    android:layout_width="120dp"
+                    android:layout_height="wrap_content"
+                    android:layout_alignParentBottom="true"
+                    android:layout_marginTop="4dp"
+                    android:layout_marginBottom="0dp"
+                    tools:srcCompat="@tools:sample/avatars" />
+
+                <TextView
+                    android:id="@+id/cast2Name"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_alignTop="@+id/cast2Img"
+                    android:layout_marginStart="130dp"
+                    android:layout_marginTop="15dp"
+                    tools:text="Cast Member Name" />
+
+                <TextView
+                    android:id="@+id/cast2Role"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_alignTop="@+id/cast2Name"
+                    android:layout_marginStart="130dp"
+                    android:layout_marginTop="20dp"
+                    tools:text="Character Name" />
+
+            </RelativeLayout>
+
+            <RelativeLayout
+                android:id="@+id/cast3"
```

Comment:
I believe you could use recycler view here to make it more flexible in case you want to add more cast info. It'll also look nicer and probably easier to manage